f/export/postgres_to_geojson/postgres_to_geojson.py

```diff
@@ -0,0 +1,136 @@
+# requirements:
+# psycopg2-binary
+
+import json
+import logging
+import os
+
+import psycopg2
+
+# type names that refer to Windmill Resources
+postgresql = dict
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def conninfo(db: postgresql):
+    """Convert a `postgresql` Windmill Resources to psycopg-style connection string"""
+    # password is optional
+    password_part = f" password={db['password']}" if "password" in db else ""
+    conn = "dbname={dbname} user={user} host={host} port={port}".format(**db)
+    return conn + password_part
+
+
+def main(
+    db: postgresql,
+    db_table_name: str,
+    storage_path: str = "/frizzle-persistent-storage/datalake/export",
+):
+    data = fetch_data_from_postgres(conninfo(db), db_table_name)
+
+    feature_collection = format_data_as_geojson(data)
+
+    save_file(feature_collection, db_table_name, storage_path)
+
+
+def fetch_data_from_postgres(db_connection_string, table_name: str):
+    """
+    Fetches all data from a specified PostgreSQL table.
+
+    Parameters
+    ----------
+        db_connection_string (str): The connection string for the PostgreSQL database.
+        table_name (str): The name of the table to fetch data from.
+
+    Returns
+    -------
+        tuple: A tuple containing a list of column names and a list of rows fetched from the table.
+    """
+
+    try:
+        conn = psycopg2.connect(db_connection_string)
+        cursor = conn.cursor()
+        cursor.execute(f"SELECT * FROM {table_name};")
+        columns = [desc[0] for desc in cursor.description]
+        rows = cursor.fetchall()
+    except psycopg2.Error as e:
+        logger.error(f"Error fetching data from {table_name}: {e}")
+        raise
+    finally:
+        cursor.close()
+        conn.close()
```

Comment:
At some point in the near future I  am going to go on an advocacy crusade for better practices around connection and cursor management. Namely upgrading to psycopg v3 and using connection  managers.

Let's not bother with that here. I'll open a separate issue and create an example.